src/Components/Component1/MoviesList.jsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react'
+import MovieCard from './MovieCard'
+import { DUMMY_JSON_PAYLOAD } from './Constants'
+import './styles.css'
+let movies = DUMMY_JSON_PAYLOAD
+
+export default function MoviesList(){
+    return (
+        <>
+        {movies.map((movie) => (
+            <div className='gallery'>
```

Comment:
every element in React list should have **key** - [documentation](https://reactjs.org/docs/lists-and-keys.html#basic-list-component)